test-suite-consul-graal/build.gradle.kts

```diff
@@ -0,0 +1,52 @@
+import java.util.Locale
+
+plugins {
+    id("org.graalvm.buildtools.native") version "0.9.14"
+    id("io.micronaut.library") version "3.7.0"
+    id("io.micronaut.test-resources") version "3.7.0"
+}
+
+repositories {
+    mavenCentral()
+}
+
+micronaut {
+    version.set(libs.versions.micronaut.asProvider())
+    testResources {
+        enabled.set(true)
```

Comment:
is this necessary? I thought test resource gradle plugin was enabled automatically once you add it.